packages/connect/src/data/coinInfo.ts

```diff
@@ -33,16 +33,17 @@ export const getBitcoinNetwork = (pathOrName: DerivationPath) => {
     return networks.find(n => n.slip44 === slip44);
 };
 
-export const getEthereumNetwork = (pathOrName: DerivationPath) => {
+export const getEthereumNetwork = (pathOrNetworkSymbol: DerivationPath) => {
     const networks = cloneObject(ethereumNetworks);
-    if (typeof pathOrName === 'string') {
-        const name = pathOrName.toLowerCase();
 
-        return networks.find(
-            n => n.name.toLowerCase() === name || n.shortcut.toLowerCase() === name,
-        );
+    if (typeof pathOrNetworkSymbol === 'string') {
+        const networkSymbol = pathOrNetworkSymbol.toLowerCase();
+
+        // check coins-eth.json for chain; chain === networkSymbol
+        return networks.find(network => network.chain.toLowerCase() === networkSymbol);
```

Comment:
TODO, if chain is `undefined` (now empty string), fall back to 
`n => n.name.toLowerCase() === networkSymbol || n.shortcut.toLowerCase() === networkSymbol `

Reply:
In `getEthereumNetwork` it will have chain all the time.